### PR TITLE
Last move nag

### PR DESCRIPTION
**Pull request by ukscid**
Here a addition option to show a NAG for the last move on the board.
ToDo: Translation

### CODE REVIEW

File: tcl/board.tcl [hunk 1]
```diff
@@ -1542,6 +1569,13 @@ proc  ::board::lastMoveHighlight {w moveuci} {
     if { $::arrowLastMove } {
         ::board::mark::DrawArrow $w.bd $square1 $square2 $::highlightLastMoveColor
     }
+    set s ""
+    regexp \[!\?]+ $nag s
+    if { $s ne "" && $::highlightLastMoveNag  } {
```
Comment (benini) [inline tcl/board.tcl]: It is better to not use one char variables and the regexp should be executed only if the option is active:
if { $::highlightLastMoveNag && [regexp {[!?]+} $nag nag] } {

File: tcl/board.tcl [hunk 2]
```diff
@@ -1528,7 +1555,7 @@ proc ::board::drawText {w sq text color args {shadow ""} } {
 }
 
 # Highlight last move played by drawing a rectangle around the two squares and/or an arrow
-proc  ::board::lastMoveHighlight {w moveuci} {
+proc  ::board::lastMoveHighlight {w moveuci {nag ""}} {
   $w.bd delete highlightLastMove
   if { ! $::highlightLastMove && ! $::arrowLastMove } {return}
```
Comment (benini) [inline tcl/board.tcl]: This line should be removed (highlighlastMoveNag does not work if the other options are all inactive).

File: tcl/windows/preferences.tcl [hunk 3]
```diff
@@ -188,11 +188,14 @@ proc ::preferences::moves { t } {
     ttk::spinbox $t.high.thick -width 2 -textvariable ::highlightLastMoveWidth -from 1 -to 5 -increment 1 \
         -validate key -validatecommand { return [string is digit %S] }
     ttk::button $t.high.color -text $::tr(ColorMarker) -command chooseHighlightColor
+    # TODO translate text
+    ttk::checkbutton $t.high.nag -variable ::highlightLastMoveNag -text "Show NAG"
```
Comment (benini) [inline tcl/windows/preferences.tcl]: -text [tr AnnotationSymbols] -command "updateBoard"

**Review by benini — COMMENTED** [review]
It looks good.
Just a few minor corrections.

**Comment by ukscid** [comment]
I insert your improvements and add the translation.
Thanks for your improvements

**Pull request merged**